core/src/test/java/nl/nn/adapterframework/jdbc/migration/MigratorTest.java

```diff
@@ -143,16 +140,6 @@ private String removeComments(String file) throws IOException {
 	}
 
 	private String applyIgnores(String sqlScript) {
-		Pattern regex = Pattern.compile("(\\d+)\\'\\)");
-		Matcher match = regex.matcher(sqlScript);
-		if(match.find()) {
-			String deploymentId = match.group(1);
-			sqlScript = sqlScript.replace(deploymentId, "IGNORE");
-		}
-		else {
-			fail("no match found");
-			return null;
-		}
```

Comment:
Weet je waarom het deployment-id niet meer wordt gezet in de tests?

Reply:
Ja, het is een beetje een apart verhaal, maar blijkbaar ging er iets fout wanneer een tabel gelocked was en het genereren van het deploymentId. Standaard wordt hij nu alleen gezet als liquibase direct naar een DB schrijft en niet meer wanneer hij naar een writer de sql wegschrijft.